src/state/persisted/legacy.ts

```diff
@@ -35,7 +35,7 @@ type LegacySchema = {
     description: string
     avatar: string
   }
-  onboarding: {
+  onboarding?: {
```

Comment:
Migration was failing bc we've removed these mobx models now. We should be careful going forward to make sure we can resume from a partial migration. So far so good, and in the next release with these changes, it will be a full migration, so probably not a big deal...